GitUI/WindowsJumpListManager.cs

```diff
@@ -23,12 +23,16 @@ public sealed class WindowsJumpListManager : IDisposable
         private ThumbnailToolBarButton _commitButton;
         private ThumbnailToolBarButton _pushButton;
         private ThumbnailToolBarButton _pullButton;
-        private bool _toolbarButtonsCreated;
+        private bool ToolbarButtonsCreated => _commitButton != null;
```

Comment:
[nit] since it has become a property we need to move it down to where the rest of properties are defined

Reply:
updated, squashed, rebased